services/mailer.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config()
+
+const sendConfirmationEmail = () => {
+    const mailgun = require('mailgun-js');
+    const DOMAIN = process.env.DOMAIN;
+    const mg = mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.DOMAIN});
+    const data = {
+        from: process.env.SENDER_EMAIL,
+        to: process.env.RECEIVER_EMAIL,
+        subject: 'Congratulations!',
+        text: 'Welcome to C0D3',
+        html: "<h1>Confirm your E-mail</h1><button>Confirm</button>"
+    };
+    mg.messages().send(data, function(error, body){
+        if(error){
+            console.log(error)
```

Comment:
we're no longer using console.log to check errors. You need to `const logger = require('../lib/log')` to get the logger. You will use `logger.error(error)` here.